### PR TITLE
add new upcoming events

### PR DESCRIPTION
**Pull request by relu91**

### CODE REVIEW

File: docs/activities/events/index.html [hunk 1]
```diff
@@ -28,12 +28,14 @@ <h2>W3C WoT – Events</h2>
 		<li>
 			<strong>Upcoming</strong>
 			<ul>
-			    <li><a href="https://iiwot24.nws.cs.unibo.it/">IEEE IIWOT 2024, January 6, 2024</a></li>
+				<li><a href="https://www.w3.org/WoT/IG/wiki/Wiki_for_WoT_Week_2024_planning">WoT Week, 25-29 November 2024</a></li>
+				<li><a href="https://www.w3.org/2024/09/TPAC/Overview.html">W3C TPAC 2024, 23–27 September 2024</a></li>
```
Comment (danielpeintner) [inline docs/activities/events/index.html:32]: Shall we flip the 2 entries, first things first ?

**Review by danielpeintner — APPROVED** [review]
Thanks.

One comment below..

File: docs/activities/events/index.html (hunk 1 above)
Comment (relu91) [reply docs/activities/events/index.html:32]: ah good point, I don't know I just followed the chronological order they would have when they are all past events...

Comment (egekorkan) [reply docs/activities/events/index.html:32]: I got confused a bit now, not sure what we were following :D For a first time visitor, the "sooner" event should indeed be on top.

**Review by danielpeintner — APPROVED** [review]

File: docs/activities/events/index.html (hunk 1 above)
Comment (danielpeintner) [reply docs/activities/events/index.html]: Great, I think we should merge *before* TPAC is over ;-)

**Pull request merged**